packages/blockchain/src/index.ts

```diff
@@ -89,7 +96,8 @@ export interface BlockchainOptions {
 export default class Blockchain implements BlockchainInterface {
   db: LevelUp
   dbManager: DBManager
-  ethash?: Ethash
+
+  _ethash?: Ethash
```

Comment:
When we implement Clique we might want to figure out how we handle this here on a more abstract matter. Otherwise, we'd also create a `_clique?: Clique` object; it would make sense to pack those together in a more common interface, like a `Consensus` class?

Reply:
Yeah, I didn't find a good abstraction for it now and found it overblown to dig further this this is just a preparatory PR for some time in the future. On the first round I now added the `_` to more clearly indicate that this is a member which shouldn't be used from the outside. Think a structure should become more clear once we start with e.g. Clique integration.

Reply:
Small remark here; if the VM only uses it in the tests then I think it is OK to just cast the `blockchain` object as any in order to get `_ethash`.